src/Serilog.Sinks.AzureDataExplorer/Durable/ExponentialBackoffConnectionSchedule.cs

```diff
@@ -0,0 +1,63 @@
+
+namespace Serilog.Sinks.AzureDataExplorer.Durable
+{
+    /// <summary>
+    /// Based on the BatchedConnectionStatus class from <see cref="Serilog.Sinks.PeriodicBatching.PeriodicBatchingSink"/>.
+    /// this class implements an exponential backoff algorithm for retrying connections.
+    /// The class has a period and a counter for the number of failures since the last successful connection.
+    /// It also provides a mechanism for marking successes and failures, and calculates the next interval to wait based on the number of failures, subject to a minimum and maximum interval.
+    /// </summary>
+    public class ExponentialBackoffConnectionSchedule
```

Comment:
We can provide some examples in README / samples as to how to do this type of a backoff.  May be we need to mention that we have the following resiliency options

Reply:
added info in README regarding this capability.